### DISCUSSION TITLE
allow song select to show multiple modes at once (all modes selection)

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20790] [Ideas] [answered]
I like to sometimes just F2 with all modes active, but this isnt possible in lazer as you are only given the selected mode, with none others. There should be a way to select all modes, such as a 5th option to enable all modes, or adding back the "mode=" operator, which would override any current selections, with "mode=a" selecting all modes

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/6500
